packages/odyssey-react/src/components/Link/Link.module.scss

```diff
@@ -46,7 +47,8 @@
   align-items: center;
 }
 
-.infobox .link {
+.infobox .link,
+.banner .link {
```

Comment:
this is starting to feel like it should be a variant 😅 : similar to how we've setup the `dismiss` Button variant?

Reply:
I believe these are the only two instances of this, if any more pop up I would be comfortable with doing that after a conversation with @joejezowski-okta !

Reply:
I agree, two separate contexts feels like that magic number to consider promoting a particular style to a variant. Three is a guaranteed promotion.

Reply:
Would there be a chance a developer can style a banner/infobox with a blue link?